app/services/i18n/i18n.ts

```diff
@@ -153,7 +153,7 @@ export class I18nService extends PersistentStatefulService<II18nState> implement
 
   setLocale(locale: string) {
     this.SET_LOCALE(locale);
-    electron.remote.app.relaunch();
+    electron.remote.app.relaunch({ args: [] });
```

Comment:
Ah interesting.  I guess by default `relaunch` will use the same args it started with currently.